src/pages/tutorial/react/step-3.mdx

```diff
@@ -274,17 +254,15 @@ Finally, if neither of those are true, it means we have our data! One nice advan
 
 We need to render the `RepoTable` in `Query`'s `return()` statement once the request is no longer loading and when there are no errors. To do so, replace the `RepoTable` line with the following `Query`.
 
-##### src/content/RepoPage/RepoPage.js
+#### src/content/RepoPage/RepoPage.js
```

Comment:
Missed one! :)

Reply:
Too many code blocks! Hopefully that's all of them :)